schema/deploy/mutations/delete_project_revision_and_amendments.sql

```diff
@@ -0,0 +1,10 @@
+-- Deploy cif:mutations/delete_project_revision_and_amendments to pg
+begin;
```

Comment:
Same comment as for the graphql mutation call above, we should probably just call this `delete_project_revision`